CI/appveyor.build.ps1

```diff
@@ -1,16 +1,21 @@
 cd "$Env:APPVEYOR_BUILD_FOLDER"
 
+. CI\appveyor.set-environment.ps1
+SetQtBaseDir "C:\src\verbose_output.log"
+SetMingwBaseDir "C:\src\verbose_output.log"
+
+git submodule update --init --recursive
```

Comment:
This should not be here IMHO - as I've noted where it was last commented out - the qmake / cmake project files specifically update _only_ the submodules needed for the platform and build options - as it is here the `--recursive` will pull in the Sparkle-glue for the macOS updater software as well as the dblsqd updater software EVEN IF the `WITH_UPDATER` is `NO` option is set previously...!

Reply:
> will pull in the Sparkle-glue for the macOS updater software as well as the dblsqd updater software EVEN IF the WITH_UPDATER is NO option is set previously...!

It wouldn't hurt in any way, but you are right, I forgot this place. Will fix it up.